extensions/amp-analytics/0.1/events.js

```diff
@@ -1279,19 +1293,32 @@ export class VideoEventTracker extends EventTracker {
   /** @override */
   add(context, eventType, config, listener) {
     const videoSpec = config['videoSpec'] || {};
-    const selector = config['selector'] || videoSpec['selector'];
-    const selectionMethod = config['selectionMethod'] || null;
-    const targetReady = this.root.getElement(
-      context,
-      selector,
-      selectionMethod
+    const selectorValue = userAssert(
+      config['selector'] || videoSpec['selector'],
+      'Missing required selector on video trigger'
+    );
+    const selectorArray = isArray(selectorValue)
+      ? selectorValue
+      : [selectorValue];
```

Comment:
nit for naming to avoid type suffix:

```suggestion
    const selectorOrSelectors = userAssert(
      config['selector'] || videoSpec['selector'],
      'Missing required selector on video trigger'
    );
    const selectors = isArray(selectorOrSelectors)
      ? selectorOrSelectors
      : [selectorOrSelectors];
```